dotCMS/src/main/java/com/dotmarketing/portlets/workflows/ajax/WfActionAjax.java

```diff
@@ -6,8 +6,10 @@
 import com.dotmarketing.business.APILocator;
 import com.dotmarketing.business.web.UserWebAPI;
 import com.dotmarketing.business.web.WebAPILocator;
+import com.dotmarketing.common.db.DotDatabaseMetaData;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.common.db.DotDatabaseMetaData'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)